djangocms_googlemap/models.py

```diff
@@ -222,6 +223,12 @@ class GoogleMapMarker(CMSPlugin):
         blank=True,
         help_text=_('Geographical longitude in degrees (e.g. "7.447446").'),
     )
+    icon = FilerImageField(
+        verbose_name=_('Icon'),
+        blank=True,
+        null=True,
+        on_delete=models.SET_NULL,
```

Comment:
it would make sense to add an info text like "A marker icon identifies a location on a map. By default, it uses a standard image from Google."